src/http-client/node-http2-client.ts

```diff
@@ -188,6 +180,14 @@ class SessionWrapper {
           .on("error", (error: any) => rejectPromise(error))
           .on("response", onResponse);
         req.write(JSON.stringify(requestData), "utf8");
+
+        // req.setTimeout must be called before req.end()
+        if (client_timeout_ms !== undefined) {
```

Comment:
this should never be undefined.

We should always have some kind of client timeout (though it wll be based on the query timeout + the buffer)